custom_components/elegoo_printer/elegoo_sdcp/server.py

```
@@ -0,0 +1,401 @@
+
+            # 2. Calculate the required values for the headers.
+            file_size = len(file_data)
+            md5_hash = hashlib.md5(file_data).hexdigest()
+            transfer_uuid = str(uuid.uuid4())
+
+            self.logger.debug(
+                f"File Size: {file_size}, MD5: {md5_hash}, UUID: {transfer_uuid}"
+            )
+
+            # 3. Construct the required headers for the printer.
+            printer_headers = {
```

Ruff (0.11.9)

Local variable `printer_headers` is assigned to but never used

Remove assignment to unused variable `printer_headers`

(F841)